prime-angular-frontend/src/app/modules/site-registration/pages/hours-operation-page/hours-operation-page.component.ts

```diff
@@ -161,11 +161,23 @@ export class HoursOperationPageComponent extends AbstractCommunitySiteRegistrati
 
   protected afterSubmitIsSuccessful(): void {
     const site = this.siteService.site;
-    let routePath = SiteRoutes.REMOTE_USERS;
-
-    if (site.careSettingCode === CareSettingEnum.COMMUNITY_PHARMACIST || site.careSettingCode === CareSettingEnum.DEVICE_PROVIDER) {
-      routePath = SiteRoutes.ADMINISTRATOR;
-    } else if (this.isCompleted) {
+    let routePath;
+
+    switch (site.careSettingCode) {
```

Comment:
Feels like this could be in an `else` since `this.isCompleted` will always override when true.